docs/local-setup.md

```diff
@@ -0,0 +1,52 @@
+## Backend 
+
+#### Start App
+1. Run `npm run init` first time to install dependencies and initialize your local database
+3. Run `npm run develop`
+4. Go to http://localhost:1337/admin and login using email `local-admin@devlaunchers.org` and password is `W&x5ZzOMtBCVt1YAUiuSzt~5`.
+
+#### Init command will do the following
+
+1. Runs NPM Install
+
+2. Builds Strapi Admin UI
+
+3. Set up default environment file into `.env`
+
+4. Starts up Strapi to allow database initialized
+
+5. Enables Google provider using test app client id and secret.
+
+6. Setup Authenticated and Public role permissions.
+
+7. Creates new Strapi admin account.
+
+8. Creates default interests and projects
+
+
+## FrontEnd
+
+#### Start App
+1. Update `env.development` to point to local backend
```

Comment:
Should we suggest cloning the frontend repo?

Reply:
@chungthuang don't think it's worth it because soon we're going to migrate to monorepo

Reply:
Added